extensions/desktop/command-chain-kde/desktop-launch

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
-if [ "$QT_VERSION" == "6" ]; then
-
-set -- "${SNAP}/kf6/snap/command-chain/desktop-launch6" "$@"
-# shellcheck source=/dev/null
-source "${SNAP}/snap/command-chain/run"
-
-elif [ "$QT_VERSION" == "5" ]; then
-
-set -- "${SNAP}/kf5/snap/command-chain/desktop-launch" "$@"
-# shellcheck source=/dev/null
-source "${SNAP}/snap/command-chain/run"
+echo $PLATFORM_PLUG
 
+if [ $PLATFORM_PLUG == "kf6-core24" ]
```

Comment:
While it's probably minimal, using these bashisms will generally result in slightly better performance (not having to start `test`) and a bit safer if for some reason the `$PLATFORM_PLUG` variable gets something weird in it:

```suggestion
if [[ "${PLATFORM_PLUG}" == "kf6-core24" ]]
```